src/twisted/trial/test/pyunitcases.py

```diff
@@ -0,0 +1,15 @@
+import unittest
```

Comment:
I think that it would help to have a top comment/docstring explaining why this needs to be a separate file.

As a code reader, the previous implementation in which this "system under test" was defined in `test_pyunitcompat.py` file, is a version that is easier to read.